src/Http/Http.Results/src/Accepted.cs

```diff
@@ -31,10 +31,7 @@ internal Accepted(string? location)
     /// <param name="locationUri">The location at which the status of requested content can be monitored.</param>
     internal Accepted(Uri locationUri)
     {
-        if (locationUri == null)
-        {
-            throw new ArgumentNullException(nameof(locationUri));
-        }
+        ArgumentNullException.ThrowIfNull(locationUri);
```

Comment:
This seems to be one place where we didn't actually need this since this is internal and we already null check in `TypedResults.Accepted(Uri)`. We don't have any tests for `ArgumentNullExceptions` thrown from the `TypedResults` methods, do we?

Reply:
[We do actually](https://github.com/dotnet/aspnetcore/blob/main/src/Http/Http.Results/test/TypedResultsTests.cs#L97).

I actually think most of these internal constructors are unused and left-over from when the types were being prepared to be made public and have public constructors. I can go through them and remove all the unused constructors